### ISSUE TITLE
Update README screenshots

### ISSUE THREAD

**Issue by fedarko** [biocore/empress#413]
With a lot of the changes over the past few weeks (default node color change, leaf sorting, defaulting to not showing node circles, ...) the README's screenshots are now fairly out-of-date. It'd be cool if we could somehow get McHelper to generate these automatically...?

**Comment by gibsramen** [comment]
Interesting idea. The parts with insets seem like they would be a major pain, though. The normal screenshots could probably be achieved with Selenium or something similar?

**Comment by fedarko** [comment]
Yeah, the insets would complicate things. A reasonable solution might just be taking two separate screenshots (one for the inset section, one for the "zoomed out" section) and then placing these next to each other in the README? It'd take some finagling to make it look right, but I think the advantage of having this automated will save a lot of time.
